tests/tck/features/optimizer/PushFilterDownLeftJoinRule.feature

```diff
@@ -13,28 +13,34 @@ Feature: Push Filter down LeftJoin rule
       LOOKUP ON player WHERE player.name=='Tim Duncan'
       | YIELD $-.VertexID AS vid
       |  GO FROM $-.vid OVER like BIDIRECT
-      WHERE any(x in split($^.player.name, ' ') WHERE x contains 'Ti')
-      YIELD $^.player.name, like._dst AS vid
-      | GO FROM $-.vid OVER like BIDIRECT WHERE any(x in split($^.player.name, ' ') WHERE x contains 'Ti')
-      YIELD $^.player.name
+      WHERE any(x in split($$.player.name, ' ') WHERE x contains 'Ti')
+      YIELD $$.player.name, like._dst AS vid
+      | GO FROM $-.vid OVER like BIDIRECT WHERE any(x in split($$.player.name, ' ') WHERE x contains 'Ti')
+      YIELD $$.player.name
       """
     Then the result should be, in any order:
-      | $^.player.name   |
-      | "Tiago Splitter" |
+      | $$.player.name |
+      | "Tim Duncan"   |
     And the execution plan should be:
       | id | name         | dependencies | operator info |
-      | 16 | Project      | 26           |               |
-      | 26 | LeftJoin     | 25           |               |
-      | 25 | Filter       | 13           |               |
-      | 13 | Project      | 22           |               |
-      | 22 | GetNeighbors | 9            |               |
-      | 9  | Project      | 24           |               |
-      | 24 | LeftJoin     | 23           |               |
-      | 23 | Filter       | 6            |               |
-      | 6  | Project      | 21           |               |
-      | 21 | GetNeighbors | 2            |               |
-      | 2  | Project      | 17           |               |
-      | 17 | IndexScan    | 0            |               |
+      | 22 | Project      | 21           |               |
+      | 21 | Filter       | 20           |               |
```

Comment:
Why not add another test case instead of changing this?
This has compromised the test of the other code logic.